config/environments/production.rb

```diff
@@ -76,4 +76,14 @@
 
   # Do not dump schema after migrations.
   config.active_record.dump_schema_after_migration = false
+
+  config.action_mailer.delivery_method = :smtp
+  config.action_mailer.smtp_settings = {
+    address:              ENV['MAILGUN_SMTP_SERVER'],
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.